@xen-orchestra/fs/src/nfs.js

```diff
@@ -1,18 +1,26 @@
 import execa from 'execa'
 import fs from 'fs-extra'
 import { forEach } from 'lodash'
+import { join } from 'path'
+import { tmpdir } from 'os'
 
 import LocalHandler from './local'
 
 const DEFAULT_NFS_OPTIONS = 'vers=3'
 
 export default class NfsHandler extends LocalHandler {
+  constructor (remote, { mountsDir = join(tmpdir(), 'xo-fs-mounts') }) {
```

Comment:
@badrAZ @pdonias I'm open to suggestions for a better name for this option. :slightly_smiling_face:

Reply:
`mountsDir` sounds good :+1: